src/AvroConvert/Features/GenerateModel/GenerateModel.cs

```diff
@@ -189,9 +190,16 @@ private void ResolveRecord(JObject parent, NetModel.NetModel model)
 
                     fieldType.Name = field["name"].ToString();
 
-                    if (field["default"] is JValue)
+                    if (field["default"] is JValue _value)
                     {
-                        fieldType.Default = field["default"].ToString();
+                        if(_value.Type != JTokenType.Float)
```

Comment:
Hi,
Thank you for the PR!
Just one question, why the behavior here is different for JTokenType.Float?

Reply:
Thanks for the comment.
Yes, it can by simplify, tests not broken